tests/docker/Dockerfile

```diff
@@ -157,6 +157,14 @@ RUN /franz-bench && \
 
 #################################
 
+FROM base as flink
+
+COPY --chown=0:0 --chmod=0755 tests/docker/ducktape-deps/flink /
+RUN /flink && \
+    rm /flink
+
+#################################
```

Comment:
Isn't this missing the actual flink install to the base image?

Example: https://github.com/redpanda-data/redpanda/pull/14818/files#diff-69679cf19b9e6a7a57703807f503d32f680101ffe6862958eb60f88eace6868aR307

Reply:
Done